src/scripts/browserDeviceProfile.js

```diff
@@ -659,6 +677,14 @@ export default function (options) {
         webmVideoCodecs.push('vp8');
     }
 
+    // WebM support is partial/unreliable from Safari versions 17.0-17.3. It was fixed in 17.4:
+    //  * https://developer.apple.com/documentation/safari-release-notes/safari-17_4-release-notes
+    const safariSupportsWebm = browser.safari && (false // eslint-disable-line sonarjs/no-redundant-boolean
```

Comment:
Please don't. It is still having problem for very long videos and I'm still experiencing player crashing on Safari TP tagged at 18.0. Please revert that.